confit/TEAM

```diff
@@ -0,0 +1,4 @@
+silver-ag
```

Comment:
The first one in the list is the team captain (the one who sends the PRs)

Reply:
ah, sorry - close this one and I'll submit it?